setup.py

```diff
@@ -234,6 +233,8 @@ def write_version(filename=os.path.join(*['airflow',
 
 devel = [
     'click==6.7',
+    'flake8-import-order>=0.18',
```

Comment:
The license of https://github.com/PyCQA/flake8-import-order is LGPL. Will this be an issue?
cc @bolkedebruin @ashb

Reply:
This may need to be a `devel-ci` extra group to make it "more" optional. If you'd rather we didn't have the extra group I can double can try and seek clarification from Apache Legal.